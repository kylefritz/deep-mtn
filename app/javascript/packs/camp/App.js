import React from 'react'
import axios from 'axios'
import * as Sentry from '@sentry/browser';
import queryString from 'query-string'
import _ from 'lodash'

export default function ShouldICamp() {
  return <h1>Should I camp? Yes!!</h1>
}
