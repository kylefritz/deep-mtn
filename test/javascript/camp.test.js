import React from 'react';
require('./configure_enzyme');
import { shallow } from 'enzyme';

import App from 'camp/App'

test('snapshot', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
