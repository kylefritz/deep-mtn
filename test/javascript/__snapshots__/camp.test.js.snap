// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<h1>
  Should I camp? Yes!!
</h1>
`;
